unix_integration/resolver/src/bin/kanidm_unixd.rs

```diff
@@ -900,7 +898,7 @@ async fn main() -> ExitCode {
                 Ok(mk) => mk,
                 Err(err) => {
                     error!(?err, "Unable to load machine root key - This can occur if you have changed your HSM pin");
-                    error!("To proceed you must remove the content of the cache db ({}) to reset all keys", cfg.db_path.as_str());
+                    error!("To proceed you must remove the content of the cache db ({}) to reset all keys", cfg.cache_db_path.as_str());
```

Comment:
I'm thinking we actually need to do something else here because the db in question will later contain things we *don't* want to just be deleted randomly.  I think for now we leave it as the db itself today still is disposable.

In the future we have to consider things like localhost kdc, which will require us to store some weakerpassword hashes. To protect that I want to use the TPM, and that then will have issues if something *changes* in the TPM, so I want to make some ways to have recover such as a backup-key or something else. That means we don't want to delete the db, we just need a way to re-key somehow. (that rekeying process is TBD).

Reply:
with `remove the content` do we mean `echo "" > [path]`? If so, may as well just print the line we want them to run. That way we don't have to worry about them trying to dump database tables or mess around with the file.